src/components/menu/test/menu.spec.ts

```diff
@@ -116,6 +116,42 @@ describe('MenuController', () => {
       expect(menu).toEqual(someMenu);
     });
 
+    it('should get the only left menu when is not rtl', () => {
+      let someMenu = mockMenu();
+      someMenu.side = 'start';
+      menuCtrl._register(someMenu);
+
+      let menu = menuCtrl.get('left');
+      expect(menu).toEqual(someMenu);
+    });
+
+    it('should get the only right menu when is rtl', () => {
```

Comment:
I am not sure how, but for this, and the `end`=`left` you need to inject `Platform`, and set direction to `rtl`
If you would run these tests, they won't pass